ai_coach/ai_coach.py

```diff
@@ -140,12 +183,22 @@ def ask_from_coach(self, data, suffix=''):
         if not data['answer']:
             return {'error': 'Answer must be required'}
 
+        if self.feedback_count >= self.feedback_threshold:
+            return {'error': 'You have reached maximum limit for asking help from coach'}
```

Comment:
mark for translations and change the text `You've exhausted all available chances to ask the coach for help`